daml-script/daml3/Daml/Script/Questions/Submit/Error.daml

```diff
@@ -18,11 +22,15 @@ data DevErrorType
 
 -- | Additional debugging info optionally provided by the ledger
 data ContractNotFoundAdditionalInfo
-  = NotFound -- The generic "this contract does not exist"
-  | NotActive -- The contract did exist, but has since been archived
+  = NotFound with -- The generic "this contract does not exist"
+      templateIdentifier : Text
```

Comment:
We don't pack this up into an AnyContractId? I guess if we're not allowing users to have this data, the untyped version might be enough - but if we ever did change that, you can't make a typed contract id from the templateId and contractId as text.